### PR TITLE
add files from toolbox-workshop/latex-template

### PR DESCRIPTION
**Pull request by chrbeckm**

### CODE REVIEW

**Comment by maxnoe** [comment]
Please also add the CI config building the template

**Comment by chrbeckm** [comment]
> Please also add the CI config building the template

Does that also get added when using the template? Might irritate some people and not be used.

**Comment by maxnoe** [comment]
Good point. But we need to check this repo somehow.

Unfortunately it seems there is no good solution for ignoring files for the template other than running a CI job that deletes the unwanted files in the newly created repo once it is created:

https://github.com/orgs/community/discussions/22183

**Comment by maxnoe** [comment]
https://github.com/orgs/community/discussions/22183#discussioncomment-6321583

File: README.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
-Dieses Repository enthält ein Template für Versuchsprotokolle, basierend auf dem PeP et al. Toolbox Workshop.
+## Vorlage für Praktikumsprotokolle
+
+Dieses Repository enthält die Struktur für Praktikumsprotokolle mit allem,
+was im Toolbox Workshop besprochen wurde.
+
+In Basisordner liegen
+- der LaTeX-Header ('header.tex'),
+- die Literaturdatenbank (`lit.bib`),
+-  sowie die der LaTeX-Header für matplotlib (`header-matplotlib.tex`)
```
Comment (aknierim) [inline README.md:12]: Unnecessary space

**Review by aknierim — APPROVED** [review]
Looks good, though!

**Review by aknierim — APPROVED** [review]

**Pull request merged**